src/vivarium/examples/boids/forces.py

```diff
@@ -59,7 +61,8 @@ def apply_force(self, index: pd.Index[int], acceleration: pd.DataFrame) -> pd.Da
             max_speed=self.max_speed,
         )
 
-        acceleration.loc[force.index, ["x", "y"]] += force[["x", "y"]]
```

Comment:
There was an interesting bug here where some boids component already included "x" and "y" in the `columns_required` property (which means those were automatically registered as AttributePipelines), but then `acceleration` is also an AttributePipeline whose source creates a dataframe including "x" and "y" columns. The end result was that we were getting _two_ columns in the state table for "x" and "y".

Reply:
gross